.gitignore

```diff
@@ -39,7 +39,7 @@ build
 
 **/.env
 !.env
-!launchers/.env
+!launchers/.env.extensions
 !connector/.env
```

Comment:
this looks like there might be an .env.broker missing. but the question is, do we actually want to ignore it? What's this line? 😄

Reply:
Yes, the broker is the line below with the previous naming: connector/.env

And we *don't* want to ignore it.

Reply:
And it should have been launchers/.env.broker